test/integrated/lsms.jl

```diff
@@ -151,9 +151,11 @@ for FT in (Float32, Float64)
             m::DummyModel3{FT},
```

Comment:
I can't add a comment on those lines but I don't think we need `make_update_aux` for DummyModel4 or DummyModel1

Reply:
How come?

Reply:
Ahh I was thinking they don't get called, but actually I think I was wrong. It'll get called for both of them in the last testset `make_set_initial_aux_state(m)`